README.md

```diff
@@ -97,6 +97,10 @@ For documentation on how to configure code-gov-front-end, go [here](CONFIGURATIO
 ## Questions?
 
 If you have questions, please feel free to [send us an email](mailto:code@gsa.gov).
+Twitter: @CodeDotGov
```

Comment:
Can you add a link to the twitter account? Here's the markdown to use:
[@CodeDotGov](https://twitter.com/CodeDotGov)